The framework build of Python on the Mac provides a pythonw for doing
graphical applications.  Also the Current symlinks needs to be managed.

--- python.eselect
+++ python.eselect
@@ -46,7 +46,7 @@
 # Try to remove python and python.1 symlinks
 remove_symlinks() {
 	local symlink symlink_target symlink_target_found
-	rm -f "${INTERPRETER_PATH}"{idle,pydoc,python,python-config} &>/dev/null && \
+	rm -f "${INTERPRETER_PATH}"{idle,pydoc,python,pythonw,python-config} &>/dev/null && \
 	rm -f "${MAN_PATH}"python.1{,.gz,.bz2,.lzma,.xz} &>/dev/null
 	for symlink in "${INTERPRETER_PATH}python"?; do
 		[[ ! -L "${symlink}" ]] && continue
@@ -58,6 +58,10 @@
 			rm -f "${symlink}"
 		fi
 	done
+
+	# Mac OS X Framework build
+	local fwdir=${INTERPRETER_PATH%/bin/}/lib/Python.framework
+	rm -f "${fwdir}"/{Headers,Python,Resources}
 }
 
 # Set a man page symlink
@@ -105,8 +105,23 @@
 		if [[ -f "${target/python/2to3-}" ]]; then
 			ln -nfs "${target/python/2to3-}" 2to3
 		fi
+		# wrapper version of python for graphical applications on Mac OS X
+		if [[ -f "${target/python/pythonw}" ]] ; then
+			ln -nfs "${target/python/pythonw}" pythonw
+		fi
 
 		popd 1>/dev/null
+
+		# Mac OS X Framework build
+		local ver=${target#python}
+		local fwdir=${INTERPRETER_PATH%/bin/}/lib/Python.framework
+		if [[ -d ${fwdir} ]]; then
+			pushd "${fwdir}" 1>/dev/null
+			ln -nfs Versions/${ver}/Headers
+			ln -nfs Versions/${ver}/Python
+			ln -nfs Versions/${ver}/Resources
+			popd 1>/dev/null
+		fi
 	else
 		die -q "Target \"${1}\" doesn't appear to be valid!"
 	fi
